src/app/pages/home/components/onboarding-steps.ts

```diff
@@ -0,0 +1,73 @@
+import AddFundsFull from '@assets/images/onboarding/steps/add-funds-light.png';
+import AddFundsFullDone from '@assets/images/onboarding/steps/add-funds-light-done.png';
+import AddFundsPopup from '@assets/images/onboarding/steps/add-funds-light-sm.png';
+import AddFundsPopupDone from '@assets/images/onboarding/steps/add-funds-light-done-sm.png';
+
+import BackUpSecretKeyFull from '@assets/images/onboarding/steps/backup-key-light.png';
+import BackUpSecretKeyFullDone from '@assets/images/onboarding/steps/backup-key-light-done.png';
+import BackUpSecretKeyPopup from '@assets/images/onboarding/steps/backup-key-light-sm.png';
+import BackUpSecretKeyPopupDone from '@assets/images/onboarding/steps/backup-key-light-done-sm.png';
+
+import BuyNftFull from '@assets/images/onboarding/steps/buy-nft-light.png';
+import BuyNftFullDone from '@assets/images/onboarding/steps/buy-nft-light-done.png';
+import BuyNftPopup from '@assets/images/onboarding/steps/buy-nft-light-sm.png';
+import BuyNftPopupDone from '@assets/images/onboarding/steps/buy-nft-light-done-sm.png';
+
+import ExploreAppsFull from '@assets/images/onboarding/steps/explore-apps-light.png';
+import ExploreAppsFullDone from '@assets/images/onboarding/steps/explore-apps-light-done.png';
+import ExploreAppsPopup from '@assets/images/onboarding/steps/explore-apps-light-sm.png';
+import ExploreAppsPopupDone from '@assets/images/onboarding/steps/explore-apps-light-done-sm.png';
+
+import { OnboardingSteps, RouteType } from '@shared/models/onboarding-types';
+import { RouteUrls } from '@shared/route-urls';
+
+export const onboardingSteps = [
```

Comment:
Should be typed as the component's props?

Is `onboardingSteps` used anywhere _other_ than looping through the components? If not, as this is a list of props, I don't see much value in abstracting to an array vs four instances of jsx below.

```tsx
<Component
  action='View secret key'
  body="Don't lose access to your account and crypto"
  event='back_up_secret_key'
  imageFull={BackUpSecretKeyFull}
  imageFullDone={BackUpSecretKeyFullDone}
  imagePopup={BackUpSecretKeyPopup}
  imagePopupDone={BackUpSecretKeyPopupDone}
  route={RouteUrls.ViewSecretKey}
  routeType={RouteType.Internal}
  title={OnboardingSteps.BackUpSecretKey}
/>
```

Doing this, we avoid: a loop, having to type an array of objects we pass in. If we have to modify or add a new item, is one easier than the other? As these onboarding steps are effectively content, it isn't repetition to add jsx explicitly.

Reply:
I think you mentioned you preferred this before, so I'll change it. 👍